dbms/src/Flash/Mpp/GRPCReceiverContext.cpp

```diff
@@ -221,6 +222,35 @@ ExchangeRecvRequest GRPCReceiverContext::makeRequest(int index) const
     return req;
 }
 
+void GRPCReceiverContext::cancelMPPTaskOnTiFlashStorageNode(LoggerPtr log)
+{
+    auto sender_task_size = exchange_receiver_meta.encoded_task_meta_size();
+    auto thread_manager = newThreadManager();
+    for (auto i = 0; i < sender_task_size; ++i)
+    {
+        auto sender_task = std::make_unique<mpp::TaskMeta>();
+        if (!sender_task->ParseFromString(exchange_receiver_meta.encoded_task_meta(i)))
+            throw Exception("parse task meta error!");
+        auto cancel_req = std::make_shared<mpp::CancelTaskRequest>();
+        cancel_req->set_allocated_meta(sender_task.release());
+        auto rpc_call = std::make_shared<pingcap::kv::RpcCall<mpp::CancelTaskRequest>>(cancel_req);
+        thread_manager->schedule(/*propagate_memory_tracker=*/true, "", [cancel_req, log, this] {
```

Comment:
`/*propagate_memory_tracker=*/false` is ok here

Reply:
Done